graphql_compiler/compiler/expressions.py

```diff
@@ -1033,6 +1104,32 @@ def to_cypher(self):
                                   left=self.left.to_cypher(),
                                   right=self.right.to_cypher())
 
+    def to_sql(self, aliases, current_alias):
+        """Return a sqlalchemy BinaryExpression representing this BinaryComposition."""
+        self.validate()
+
+        translation_table = {
+            u'=': python_operator.__eq__,
+            u'!=': python_operator.__ne__,
+            u'<': python_operator.__lt__,
+            u'>': python_operator.__gt__,
+            u'<=': python_operator.__le__,
+            u'>=': python_operator.__ge__,
+            u'&&': sql.expression.and_,
+            u'||': sql.expression.or_,
+            u'has_substring': sql.operators.ColumnOperators.contains,
+            # in_collection and not_in_collection are implemented using contains and not_contains
```

Comment:
doesn't this mean that `in_collection` and `not_in_collection` should still exist in this dict but be mapped to `contains` and `not_contains` instead? I think right now those cases would end up in the `NotImplementedError` on line 1126.

Reply:
What I meant was IR generation converts an `in_collection` to a `contains`, so an `in_collection` in the query is a `contains` in the IR. You can see that even the `MATCH` backend doesn't bother to define `in_collection`. I will clarrify.